torchaudio/backend/utils.py

```diff
@@ -58,6 +58,9 @@ def set_audio_backend(backend: Optional[str]):
     elif backend == 'sox_io':
         module = sox_io_backend
     elif backend == 'soundfile':
+        warnings.warn(
+            '"soundfile" backend is planned to change the function signatures in 0.8.0. '
+            'Please refer to https://github.com/pytorch/audio/issues/903 for the migration step.')
```

Comment:
the real issue behind this particular comment is the return type? then it could be specialized to return type?

Reply:
how about a parameter that controls the return type?